### PR TITLE
Added recipe for backports.ssl

### PR DESCRIPTION
**Pull request by pmlandwehr**
Per [`backports.ssl_match_hostname`](https://github.com/conda-forge/staged-recipes/pull/1013), I've just called this module `ssl`. That can be changed...

### CODE REVIEW

File: recipes/ssl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{%set name = "ssl" %}
```
Comment (jakirkham) [inline recipes/ssl/meta.yaml]: Let's call it `backports.ssl`. There is an `ssl` package for Python too. So this is a bit confusing if we name it as is.

File: recipes/backports.ssl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+test:
+  imports:
+    - backports
+    - backports.ssl
+    - backports.ssl.monkey
+
+about:
+  home: https://github.com/alekstorm/backports.ssl
+  license: MIT
+  # license_file: NO MANIFEST in package
```
Comment (jakirkham) [inline recipes/backports.ssl/meta.yaml]: Proposed in PR ( https://github.com/alekstorm/backports.ssl/pull/20 ).

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

**Comment by ocefpaf** [comment]
@pmlandwehr feel free to ping me on gitter too when I miss the first call.

**Comment by pmlandwehr** [comment]
Will do.

**Pull request merged**